test/test_distributions.py

```diff
@@ -1873,17 +1895,20 @@ def test_cauchy_shape_tensor_params(self):
         self.assertEqual(cauchy.log_prob(Variable(torch.ones(2, 1))).size(), torch.Size((2, 2)))
 
     def test_dirichlet_shape(self):
-        dist = Dirichlet(variable([[0.6, 0.3], [1.6, 1.3], [2.6, 2.3]]))
+        dist = Dirichlet(variable([[0.6, 0.1], [0.3, 0.3], [0.1, 0.6]]), validate_args=True)
```

Comment:
The params should not lie on the simplex; could you please revert.

Reply:
Sorry about that.